### PR TITLE
⭐️ support PKIX certificate Subject Alternative Name extension

### PR DESCRIPTION
**Pull request by chris-rock**
```
cnspec shell host google.com    
cnspec> tls.certificates { sanExtension { * }}
cnspec> tls.certificates { sanExtension { * }}
tls.certificates: [
  0: {
    sanExtension: {
      uris: []
      extension: pkix.extension id = 5842ac625349147af543f8049f60497ca270c0412667bbeb1042482e805069f9:2.5.29.17
      emailAddresses: []
      dnsNames: [
        0: "*.google.com"
        1: "*.appengine.google.com"
        2: "*.bdn.dev"
        3: "*.origin-test.bdn.dev"
        4: "*.cloud.google.com"
        5: "*.crowdsource.google.com"
        6: "*.datacompute.google.com"
        7: "*.google.ca"
        8: "*.google.cl"
        9: "*.google.co.in"
        10: "*.google.co.jp"
        11: "*.google.co.uk"
        12: "*.google.com.ar"
        13: "*.google.com.au"
        14: "*.google.com.br"
        15: "*.google.com.co"
        16: "*.google.com.mx"
        17: "*.google.com.tr"
        18: "*.google.com.vn"
...
      ]
    }
  }
  1: {
    sanExtension: null
  }
  2: {
    sanExtension: null
  }
]
```

### CODE REVIEW

File: providers/network/resources/network.lr [hunk 1]
```diff
@@ -236,6 +238,20 @@ pkix.extension {
   value string
 }
 
+// x509 certificate PKIX Subject Alternative Name extension
```
Comment (misterpantz) [inline providers/network/resources/network.lr:241]:
```suggestion
// x509 Certificate PKIX Subject Alternative Name Extension
```
for consistency :)

Comment (chris-rock) [reply providers/network/resources/network.lr:241]: Can you please align with @tas50. He changed some titles to be lower case last time https://github.com/mondoohq/cnquery/pull/2372/commits/68231c7062e22c5f207eae04b529ca9344fd1500

**Review by arlimus — APPROVED** [review]
Kudos for the addition!

File: providers/network/resources/network.lr (hunk 1 above)
Comment (arlimus) [reply providers/network/resources/network.lr:241]: I'll merge this for now. + 1 to figuring this out, I'm curious to better understand it too. I don't think I'm the right person to cast the decision on this

Comment (misterpantz) [reply providers/network/resources/network.lr:241]: This is my mistake.

**Pull request merged**